great_expectations/core/usage_statistics/anonymizers/datasource_anonymizer.py

```diff
@@ -143,19 +134,19 @@ def anonymize_simple_sqlalchemy_datasource(self, name, config):
         return anonymized_info_dict
 
     def get_parent_class(self, config) -> Optional[str]:
-        return self._get_parent_class(
+        return super().get_parent_class(
             classes_to_check=self._ge_classes + self._legacy_ge_classes,
             object_config=config,
         )
 
     def get_parent_class_v2_api(self, config) -> Optional[str]:
-        return self._get_parent_class(
+        return super().get_parent_class(
             classes_to_check=self._legacy_ge_classes,
             object_config=config,
         )
 
     def get_parent_class_v3_api(self, config) -> Optional[str]:
-        return self._get_parent_class(
+        return super().get_parent_class(
             classes_to_check=self._ge_classes,
             object_config=config,
         )
```

Comment:
The datasource anonymizer is a tricky one - we need to ensure we track both v2 and v3, meaning that `self._ge_classes` actually holds some value.

I'm leaving this anonymizer alone for the time being.